### PR TITLE
Add correct path to the scripts directory

### PR DESCRIPTION
**Pull request by nrb**
Addresses #259

### CODE REVIEW

File: scripts/upgrade.sh [hunk 1]
```diff
@@ -16,8 +16,8 @@
 # (c) 2015, Nolan Brubaker <nolan.brubaker@rackspace.com>
 set -eux pipefail
 
-OSAD_DIR='/opt/rpc-openstack/os-ansible-deployment'
-RPCD_DIR='/opt/rpc-openstack/rpcd'
+BASE_DIR="$(pwd)"
```
Comment (mancdaz) [inline scripts/upgrade.sh]: This is brittle.  The following is more deterministic:

BASE_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd ../ && pwd )

Comment (odyssey4me) [inline scripts/upgrade.sh]: or perhaps `BASE_DIR=$(dirname ${0})/../)` ?

Comment (mancdaz) [inline scripts/upgrade.sh]: Not sure that even works. You could though shorten the first example to:

```BASE_DIR=$( cd "$( dirname ${0} )" && cd ../ && pwd )```

Comment (nrb) [inline scripts/upgrade.sh]: Ok, thanks, I will revisit this then.

**Comment by nrb** [comment]
@mancdaz, @odyssey4me PR updated.

**Comment by mancdaz** [comment]
:+1:

**Comment by prometheanfire** [comment]
:+1:

**Pull request merged**